.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+env:
+  TARGET: linux/amd64
+  BUILD_FLAGS: --no-cache
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Need tags for Makefile logic to work
+          fetch-depth: 0
+      - name: Build the Docker images
+        run: make images
+  scan:
+    needs: [ "build" ]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Need tags for Makefile logic to work
+          fetch-depth: 0
+
+      - name: Build the Docker image for Trivy
```

Comment:
minor detail but should probably update Trivy to Docker Workflows

Reply:
Removed the trivy reference.